bootstrap/api/transport.go

```diff
@@ -88,13 +88,13 @@ func MakeHandler(svc bootstrap.Service, reader bootstrap.ConfigReader, logger lo
 		encodeSecureRes,
 		opts...))
 
-	r.Put("/things/state/:id", kithttp.NewServer(
+	r.Put("/things/state/:stateId", kithttp.NewServer(
```

Comment:
```suggestion
	r.Put("/things/state/:thingId", kithttp.NewServer(
```

Reply:
Its endpoint uses the struct below
`type changeStateReq struct {
	token string
	id    string
	State bootstrap.State  }`

in  bootstrap/api/requests.go
Doesn't this suggest the id is for the state?

Reply:
The endpoint is to change the state of a thing. Have a look at the service definition to where that endpoint goes.

Reply:
okay